src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const BASIC_PRICE = 40;
+  const LONG_PERIOD = 7;
+  const SHORT_PERIOD = 3;
+  const LONG_PERIOD_DISCOUNT = 50;
+  const SHORT_PERIOD_DISCOUNT = 20;
+  const totalPrice = days * BASIC_PRICE;
```

Comment:
I think there should be free space between function constants and the results of calculations.